### PR TITLE
Aw add accessibility

### PR DESCRIPTION
**Pull request by allisonjw**
#### What's this PR do?
- Wraps all inputs in labels
- Adds for attribute in label with a value of the input's id
- Gives all inputs id's and changes in scss and index.js

#### Where should the reviewer start?
- Start in the DOM to see the changes
- Most changes are in HTML to make app 100% accessible

#### How should this be manually tested?
- Test in the DOM and run accessibility audit to see our 100% score

#### Any background context you want to provide?
- Score was at 70% before changes

#### Questions:
- We will still need to test cross-browser compatibility (firefox, safari)

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -184,8 +184,10 @@ form {
         }
     }
     button {
+      align-self: center;
       background-color: grey;
       border-radius: 3px;
+      height: 2em;
```
Comment (trishalanglois) [inline src/css/base.scss:189]: Nice job using ems to make it responsive!

File: src/index.html [hunk 2]
```diff
@@ -43,34 +43,43 @@ <h3>
...
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="18000" placeholder="#">
-          <label>Minutes Active</label>
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
-          <label>Flights of Stairs</label>
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
+          <label for="form__control--date1">Date
+          <input aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
+          </label>
+          <label for="form__control--steps">Number of Steps
+          <input aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
```
Comment (trishalanglois) [inline src/index.html:50]: Do we need to add a similar pattern attribute to our splash page input for the user to enter their id?

File: src/index.html [hunk 3]
```diff
@@ -43,34 +43,43 @@ <h3>
...
           <button type="submit">Submit</button>
         </form>
       </article>
       <article class="addHydration__article">
         <form method="POST">
-            <label>Date</label>
-            <input aria-required="true" class="form-control" type="text" name="date" placeholder="YYYY/DD/MM">
-            <label>Ounces of Water Drank</label>
-            <input aria-required="true" class="form-control" type="number" name="number" pattern="[-+]?[0-9]*[.,]?[0-9]+" step=0.01 min="1" max="200" placeholder="#">
+            <label for+="form__control--date2">Date
```
Comment (cammac60) [inline src/index.html]: Small typo here, there is a + before the =

**Review by cammac60 — CHANGES_REQUESTED** [review]
Other than the typo I commented on this looks really good!

**Pull request merged**